src/main/java/hu/dungeonhunter/characters/monsters/Goblin.java

```diff
@@ -8,22 +9,35 @@
 
     @Setter
     @Getter
-    private int hp;
+    private int finalAccuracy;
```

Comment:
You're overthinking and over using class variables. Final accuracy comes from a calculation based on the base accuracy. You don't have to store this calculation result here, you need to pass the result in the related method.
The same goes for finalInitiation. Please remove these from all the character classes and make the calculator methods return the result for Fight class.

Reply:
I wanted to use this setter for a test, but in the end I didn’t use it and forgot there. Thank you, i deleted it.